api/src/main/java/at/helpch/chatchat/api/format/BasicFormat.java

```diff
@@ -0,0 +1,7 @@
+package at.helpch.chatchat.api.format;
+
+/**
+ * Represents a basic {@link Format}. These are usually used for Console formats and Private Message formats.
+ */
+public interface BasicFormat extends Format {
```

Comment:
is this interface necessary? 
cant we remove this class and rename the BasicFormatImpl implementation to just BasicFormat in the plugin module (SimpleFormat might be a better name imo too).